docs/how-to/lock-contexts-with-clerk.md

```diff
@@ -1,5 +1,16 @@
 
-## Lock context with Next.js and Clerk
+# Locking context with Next.js and Clerk
+
+This how-to guide shows you how to lock context if you're using [Clerk](https://clerk.com/) with Next.js.
```

Comment:
Is this use of 'how to lock' right? I was under the impression it should be like 'how to use lock context'.

Reply:
Yeah I agree with @yujiyokoo. 

On the other hand, I think there's some more fundamental things we need to address in this how-to:

- what are the learning outcomes for this how-to
- what is context locking
- what capabilities context locking gives you
- simplify (and ideally remove) lock context jargon

I don't think fixing any of that should be a blocker for merging this PR. We can tackle that in a separate PR.